### PR TITLE
Add link to code of conduct / how it works, from CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Leemoonsoo**
### What is this PR for?
Adding code of conduct link http://www.apache.org/foundation/policies/conduct.html to our contribution guide, as well as http://www.apache.org/foundation/how-it-works.html

### What type of PR is it?
Improvement

### Todos
* [x] - Add link to Code of conduct
* [x] - Add link to the Apache way.

### Is there a relevant Jira issue?
no

### How should this be tested?
N/A

### Screenshots (if appropriate)
N/A

### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no
* Does this needs documentation? no

### CODE REVIEW

**Comment by jongyoul** [comment]
LGTM

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ Contributing to Zeppelin (Source code, Documents, Image, Website) means you agre
 2. If not, create a ticket describing the change you're proposing in the [Jira issue tracker](https://issues.apache.org/jira/browse/ZEPPELIN)
 3. Contribute your patch via Pull Request.
 
+Before you start, please read the [Code of Conduct](http://www.apache.org/foundation/policies/conduct.html) carefully, familiarise yourself with it and refer to it whenever you need it.
+
+For those of who are not familiar with Apache project, understanding how [How it works](http://www.apache.org/foundation/how-it-works.html) would be quite helpful.
```
Comment (AhyoungRyu) [inline CONTRIBUTING.md]: @Leemoonsoo How about deleting "how" ?( i mean the former one )  
Except this, LGTM!

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -7,6 +7,10 @@ Contributing to Zeppelin (Source code, Documents, Image, Website) means you agre
 2. If not, create a ticket describing the change you're proposing in the [Jira issue tracker](https://issues.apache.org/jira/browse/ZEPPELIN)
 3. Contribute your patch via Pull Request.
 
+Before you start, please read the [Code of Conduct](http://www.apache.org/foundation/policies/conduct.html) carefully, familiarise yourself with it and refer to it whenever you need it.
```
Comment (corneadoug) [inline CONTRIBUTING.md]: familiarise - > familiarize

File: CONTRIBUTING.md (hunk 1 above)
Comment (corneadoug) [inline CONTRIBUTING.md]: For those of who are not -> For those of you who are not

And like it was said, there is a double how in that sentence

**Comment by Leemoonsoo** [comment]
@jongyoul @AhyoungRyu @corneadoug Thanks for the review. I have addressed the comment.

**Comment by Leemoonsoo** [comment]
CI failing is irrelevant. Merge if there're no more discussions.

**Pull request closed without merging**